pkgs/applications/networking/cluster/temporalite/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "temporalite";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "temporalio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-rLEkWg5LNVb7i/2IARKGuP9ugaVJA9pwYbKLm0QLmOc=";
+  };
+
+  vendorSha256 = "sha256-vjuwh/HRRYfB6J49rfJxif12nYPnbBodWF9hTiGygS8=";
+
+  # The test suite currently involves integration tests that can't be
+  # reproduced in Nix.
+  doCheck = false;
```

Comment:
Can we disable just those tests? See https://github.com/NixOS/nixpkgs/blob/2f397f1cca64859ad8897cd66a891e6d6ea3134c/pkgs/tools/text/goawk/default.nix#L19-L23 for an example